doc/content/integrations/cloud-integrations/uib/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "UIB"
+description: ""
+weight: 
+aliases: ["/integrations/cloud-integrations/uib/tts-setup"]
```

Comment:
There's no alias needed when this page isn't moved. We use aliases to make sure that old links are still working.